src/components/FlowList.vue

```diff
@@ -0,0 +1,158 @@
+<template>
+  <div class="flow-list">
+    <p-layout-table sticky>
+      <template #header-start>
+        <slot name="header-start">
+          <div class="flow-list__header-start">
+            <ResultsCount v-if="selected.length == 0" :label="localization.info.flow" :count="flowsCount" />
+            <SelectedCount v-else :count="selected.length" />
```

Comment:
I really want to combine these components into something like this where the `selected` prop is optional. 

```
<ResultsCount :count="10" :selected="4" :label="localization.info.flow" />
```

Reply:
Nice idea, [tracked here](https://github.com/PrefectHQ/prefect-ui-library/issues/1350)